### ISSUE TITLE
New-MgUserAuthenticationPhoneMethod - Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by andrewmatveychuk** [microsoftgraph/msgraph-sdk-powershell#368]
When I try to execute something like:
`New-MgUserAuthenticationPhoneMethod -UserId $_.User.Id -PhoneNumber $_.ParsedMobilePhoneNumber -PhoneType 'mobile'`
I get the following error:
`Object reference not set to an instance of an object.`

The same error is with:
`Update-MgUserAuthenticationPhoneMethod -PhoneAuthenticationMethodId $_.AuthenticationPhoneMethod.Id -UserId $_.User.Id -PhoneNumber $_.ParsedMobilePhoneNumber`

Both commands used to work in Microsoft.Graph SDK v.0.7.1, but now they fail in v.0.9.1.

The execution context:
```
Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Microsoft Windows 10.0.19041
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by andrewmatveychuk** [comment]
When adding an empty `-AdditionalProperties @{}` attribute to the commands, the error message becomes more descriptive:
`{"Message":"No HTTP resource was found that matches the request URI 'https://mface.windowsazure.com/odata/users('45de371f-3f37-4463-b9a0-defa64956c66%4033a2a1d0-2508-4bd4-93f4-debda0352afc')/authentication/phoneMethods('3179e48a-750b-4051-897c-87b9720928f7')'.","MessageDetail":"No type was found that matches the controller named 'users'."}`

**Comment by MIchaelMainer** [comment]
Will be fixed in 0.9.2.

**Comment by peombwa** [comment]
This has be fixed in v1.0,  `Install-Module Microsoft.Graph -AllowClobber -Force`.

``` ps
New-MgUserAuthenticationPhoneMethod -UserId $userId -PhoneNumber $phoneNumber -PhoneType mobile
```
Since this is a beta API, you will need to switch to beta profile in order for you to use the command. The SDK loads v1.0 profile by default.
```ps
Select-MgProfile beta
```
